egs/wham/ConvTasNet/local/prepare_data.sh

```diff
@@ -25,7 +25,7 @@ wait
 echo "Run python scripts to create the WHAM mixtures"
 # Requires : Numpy, Scipy, Pandas, and Pysoundfile
 cd $out_dir/wham_scripts
-ln -sf ../../../corpus .
+ln -sf $(realpath $wav_dir/../) .
```

Comment:
this is because when you getting into the folder, you lost the path at higher level so it needs some absoulte path soft linking to simply make the code work. But this is for my case only (on grid5k). If it does not looks right, I will delete them.